### ISSUE TITLE
"Add previous term" button should hide on friend's profile 

### ISSUE THREAD

**Issue by ccqi** [UWFlow/rmc#209]
When using flow, I realized that when I am on my friend's profile the "Add previous term" button is still showing. However, the "Add term" button automatically hides in this situation. 

This should be fixed to ensure consistency with the "Add term" button and for more intuitive UI for the user.

**Comment by JGulbronson** [comment]
If you look in `server/profile.py`, you'll see we send up the `profile_user_id` and `current_user_id`. This could be used to prevent the "Add term" button from rendering, if they aren't equal. *Super* easy fix, good first PR.

**Comment by mcyee** [comment]
I'd like to look into this!

**Comment by JGulbronson** [comment]
That'd be great! Do you need any help getting your dev environment set up?

**Comment by mcyee** [comment]
yes please! I've never really contributed to an existing project before :)

**Comment by divad12** [comment]
W00t!

Happy to help out for a half-hour if you're still stuck on anything.

**Comment by divad12** [comment]
You can shoot me an email at davidhu91@gmail.com

**Comment by JGulbronson** [comment]
Hey @mcyee, this must've gotten buried in my inbox, sorry about that!

David's definitely a great guy to help. If you'd like, you can also hop on to our Hipchat channel, that's a good place to find one of us :) You can also always message me on Facebook, or email me at jgulbronson@gmail.com

Thanks for your interest in contributing!

**Comment by mcyee** [comment]
hey @divad12, @JGulbronson, sorry for the extended delay. I've been buried under midterms and assignments, and finally have a breather. I'll look for you in HipChat :)

**Comment by divad12** [comment]
Cool!

I'm no longer on the HipChat. You can email me or add me on Facebook and
chat. Or start a Slack channel for us. :)
